ArmEmulator/Mem.fs

```diff
@@ -440,9 +418,10 @@ module Mem
 
 
 
-    type ReturnInstr = LabelO of Result<LabelInstr,string> 
-                        | MemO of Result<MemInstr,string> 
-                        | AdrO of Result<ADRInstr,string>
+    type ReturnInstr = 
```

Comment:
I'm not 100% sure this will be compatible with the other modules, because of the Results. Mine is:
```fsharp
    type ReturnInstr = 
        | MemI of MultMemInstr 
        | BranchI of BranchInstr
        | EndI of EndInstr
```